config/config.go

```diff
@@ -33,11 +33,12 @@ func init() {
 
 //ShipperConfig defines the options present in the config file
 type ShipperConfig struct {
-	Log  logp.Config `config:"logging"`
-	TLS  bool        `config:"tls"`
-	Cert string      `config:"cert"` //TLS cert file, if TLS is enabled
-	Key  string      `config:"key"`  //TLS Keyfile, if specified
-	Port int         `config:"port"` //Port to listen on
+	Log   logp.Config `config:"logging"`
+	TLS   bool        `config:"tls"`
+	Cert  string      `config:"cert"`  //TLS cert file, if TLS is enabled
+	Key   string      `config:"key"`   //TLS Keyfile, if specified
+	Port  int         `config:"port"`  //Port to listen on
+	Queue *config.C   `config:"queue"` //Shipper queue Settings
```

Comment:
Can this be an explicit type rather than using `ucfg`? Just making a `QueueConfig` struct with the fields you need for now is fine (looks like maybe just `Metrics`?), other config fields can be added as needed. I just want to make sure we keep things in explicit structs as much as possible so we don't lose track of what fields are in use the way we did in beats.

Reply:
Yah, I put this in as a placeholder while I was working with the dummy queue, we might want to remove it entirely until we have an actual queue hooked up.